tla-pp/src/test/scala/at/forsyte/apalache/tla/pp/TestConstAndDefRewriter.scala

```diff
@@ -43,19 +42,23 @@ class TestConstAndDefRewriter extends AnyFunSuite with BeforeAndAfterEach {
       root.declarations match {
         case Seq(n, overrideN: TlaOperDecl, rest @ _*) =>
           val typedN = n.withTag(Typed(IntT1()))
-          val overrideTag = Typed(OperT1(Seq(), IntT1()))
-          val typedOverrideN = TlaOperDecl(overrideN.name, List(), overrideN.body.withTag(Typed(IntT1())))(overrideTag)
+          val typedOverrideN: TlaOperDecl =
+            tla.decl(
+                overrideN.name,
+                tla.useTrustedEx(overrideN.body.withTag(Typed(IntT1()))),
+            )
           Seq(typedN, typedOverrideN) ++ rest
       }
 
-    val input = new TlaModule(root.name, newDeclarations)
+    val input = TlaModule(root.name, newDeclarations)
 
     val rewritten = new ConstAndDefRewriter(new IdleTracker())(input)
     assert(rewritten.constDeclarations.isEmpty) // no constants anymore
     assert(rewritten.operDeclarations.size == 2)
-    val expected_n = TlaOperDecl("n", List(), tla.int(10))
+    val expected_n: TlaOperDecl = tla.decl("n", 10)
```

Comment:
`expected_n : TlaOperDecl` coerces the implicit cast from `TBuilderItnernalState[OperDecl]`. You need to do this, because the next line does `expected_n == SomeOperDecl`. Typically, this conversion happens automatically if you're using `expected_n` in a place where only a `TlaOperDecl` fits, but `==` is `(Any, Any) => Bool`, so the implicit conversion wouldn't happen.